### ISSUE TITLE
[Expert] Crash from Lootr on world load after updating from 1.0.0 -> 1.1.2

### ISSUE THREAD

**Issue by mcarreon** [EnigmaticaModpacks/Enigmatica6#4578]
### Modpack Version

1.1.2

### Describe your issue.

Updated from ver 1.0.0 to 1.1.2 and get a consistent crash on world load with the basic error msg: 

```
java.lang.IllegalStateException: Unable to create backup for Lootr data files!
```

### Crash Report

https://gist.github.com/mcarreon/e24532277701a0e44c5bcf0ba1216b84

### Latest Log

https://gist.github.com/mcarreon/4d36c573257894b58de223d3928f10be

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by theboo** [comment]
Reported here:
https://github.com/noobanidus/Lootr/issues/88

**Comment by noobanidus** [comment]
Okay, so this is problematic and frustrating. I'm unable to replicate this in my local dev environment, but I've made a few changes that I think might help. I've made [this build (`12.41-alpha`)](https://www.curseforge.com/minecraft/mc-mods/lootr/files/3736564) which includes the fix from `12.41` but refactors and removes some code that's potentially causing the "another process already using file".

I'd really appreciate feedback on this build -- but if it's still failing and you're desperate to play your world, there's [this build (`12.41-alpha-nobackup`)](https://www.curseforge.com/minecraft/mc-mods/lootr/files/3736575) which completely removes the back-up creation. Ideally, I'd like to get to the bottom of the issue, as this is the second time that it's been reported since Enigmatica updated (although this code has been in Lootr releases for over a month).

If you can ping me via the Enigmatica discord (`nooby#0666`) or drop by my discord/send me a DM with whatever outcomes you have, that would be great.

**Comment by noobanidus** [comment]
This is fixed as of Lootr [`12.42`](https://www.curseforge.com/minecraft/mc-mods/lootr/files/3735664).
